src/components/Dashboard/TodoList.vue

```diff
@@ -643,7 +643,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
   enableCheckbox: Array<any> = []
   confirmEnabled = false
   panel: number = null // currently expanded panel
-  checkTimer: number = null
+  checkTimer = null // may be type number or NodeJS.Timeout
```

Comment:
This is type `any` by default.

When run locally, checkTimer is type NodeJS.Timeout. When run in the browser, it's type number. They both work the same way but this fixes some lint warnings.